arch/lkl/kernel/atomic.c

```diff
@@ -0,0 +1,86 @@
+#include <linux/kernel.h>
```

Comment:
I don't see any nicer way to do it and we will need them anyway for SMP support. We may need a different set for SMP, but we can change that later.